File: myapp/child/ChildWorkflow.cs
using sfn_ut.sfn;

namespace sfn_ut.myapp.child;

public class ChildWorkflow : SfnWorkflow<ChildPayload> {
  public ChildWorkflow() : base(new ErrorHandler()) {
    // Do nothing
  }

  public override ChildPayload Run(ChildPayload payload) {
    payload = Lambda(new Step10_Lambda(), payload);
    payload = Lambda(new Step20_Lambda(), payload);

    return payload;
  }
}

File: java-sfn-unit-test/src/main/java/com/sfnut/myapp/child/ChildWorkflow.java
package com.sfnut.myapp.child;

import com.sfnut.sfn.SfnWorkflow;

public class ChildWorkflow extends SfnWorkflow<ChildPayload> {
    public ChildWorkflow() {
        super(new ErrorHandler());
    }

    @Override
    public ChildPayload run(ChildPayload payload) {
        payload = lambda(new Step10_Lambda(), payload);
        payload = lambda(new Step20_Lambda(), payload);

        return payload;
    }
}
